extensions/answer_summarizers/models.py

```diff
@@ -239,3 +291,42 @@ def calculate_from_state_answers_dict(self, state_answers_dict):
             state_answers_dict['interaction_id'],
             self.id,
             categorized_answer_frequency_lists)
+
+
+class TopNUnresolvedAnswersByFrequency(BaseCalculation):
+    """Calculation for the top unresolved answers by frequency
+    The output from this calculation is a ranked list of unresolved answers, by
+    frequency.
+    """
+
+    def calculate_from_state_answers_dict(self, state_answers_dict):
+        """Filters unresolved answers and then computes the number of
+        occurrences of each unresolved answers.
+
+        This method is run within the context of MapReduce job.
+
+        Args:
+            state_answers_dict: dict. A dict containing state answers and
+                exploration information.
+
+        Returns:
+            stats_domain.StateAnswersCalcOutput. A calculation output
+                domain object.
```

Comment:
Explain precisely what this represents, not just its type. Maybe something like: "A calculation output object containing the top unresolved answers (up to at most `limit` answers).", or similar.

Reply:
Done